evm_body.yaml

```diff
@@ -480,13 +480,12 @@ eth_estimateGas_550:
               - $ref: '#/blockNumber_or_blocktag_param_eth'
                 title: Block Number or Block Tag
           prefixItems:
-            oneOf:
-              - allOf:
-                  - $ref: '#/transaction_object'
-                    title: Transaction Object
-                  - $ref: '#/gas_550'
-              - $ref: '#/blockNumber_or_blocktag_param_eth'
-                title: Block Number or Block Tag
+            - allOf:
+                - $ref: '#/transaction_object'
+                - $ref: '#/gas_550'
+              title: Transaction Object
```

Comment:
@SahilAujla we want this title for the transaction_object

gas_550 is a separate param right?

Reply:
@elanh we're combining the tx object and gas_550 object into one object here so the title is for both.